src/iamclient/iamclient.cpp

```diff
@@ -83,6 +83,12 @@ Error IAMClient::Stop()
     mClockSync->Unsubscribe(*this);
     mNodeInfoProvider->UnsubscribeNodeStatusChanged(*this);
 
+#ifndef CONFIG_ZTEST
```

Comment:
why ifndef required?

Reply:
for mCertHandler :)
It's removed from members if CONFIG_ZTEST is defined